gnucash/gnome/gnc-plugin-page-report.cpp

```diff
@@ -1788,17 +1787,37 @@ gnc_plugin_page_report_export_cb (GSimpleAction *simple,
                                      "document object with export-string or "
                                      "export-error."));
         }
-        result = TRUE;
     }
     else
-        result = gnc_html_export_to_file (priv->html, filepath);
-
-    if (!result)
     {
-        const char *fmt = _("Could not open the file %s. "
-                            "The error is: %s");
-        gnc_error_dialog (parent, fmt, filepath ? filepath : "(null)",
-                          strerror (errno) ? strerror (errno) : "" );
+        SCM chart_report = scm_call_1 (scm_c_eval_string ("gnc:report-chart?"),
+                                       priv->cur_report);
+        SCM html_export = (scm_is_true (chart_report)) ? SCM_BOOL_T : SCM_BOOL_F;
```

Comment:
lol, use `chart_report` directly...

Reply:
LOL indeed. Possibly all that SCM conversion gymnastics causing my brain to shut off...